macros/src/main/scala/packaged/insidePackageObject/package.scala

```diff
@@ -2,7 +2,7 @@ package packaged
 
 import scala.gestalt._
 
-package object macros2 {
+package object insidePackageObject {
   inline def plus(a: Int, b: Int): Int = meta {
     q"$a + $b"
   }
```

Comment:
I am not sure if it is an issue with dotty or the changes to enable macros. Initially I thought it was the extra method that was causing the problem, but now I see it is exclusive to macro definitions.
```scala
package packaged
package object macros {
  inline def plus(a: Int, b: Int): Int = a + b
}
```
this creates a method `plus` inside module class `package$` inside package `packaged.macros`
```scala
package packaged {
   import scala.gestalt._
   package packaged.macros {
     final lazy module val package: packaged.macros.package$ = 
       new packaged.macros.package$()
     final module class package$() extends Object() { 
       this: packaged.macros.package.type =>   
       @a.+(b) inline def plus(a: Int, b: Int): Int = a.+(b)
//...
}}
```
However:
```scala
package packaged
package object macros {
  inline def plus(a: Int, b: Int): Int = meta {
    q"$a + $b"
  }
}
```
macro definition creates a method `plus` inside module class `macros$` inside package `packaged`.
```scala
package packaged {
  import scala.gestalt._
  final lazy module val macros: packaged.macros$ = new packaged.macros$()
  final module class macros$() extends Object() { this: packaged.macros.type => 
    def plus(a: Int, b: Int): Int = ???
//...
}}
```